src/utils/rebass.ts

```diff
@@ -0,0 +1,13 @@
+import { keys } from './keys';
+
+const MRE = /^m[trblxy]?$/;
+
+const getProps = (test: (key: string) => boolean) => (props: any) =>
```

Comment:
Don't use `any`

Reply:
Not sure if there is a great type for this 🤔 See Slack discussion